tests/multihop-basic.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 set -eux
 
-export LABPATH=./
-export BABELPATH=../babeld
+BABELPATH=${BABELPATH:=../babeld}
```

Comment:
is there an advantage to declaring variables this way? Something about external overrides probably.

Reply:
exactly, this part lets you do `sudo BABELPATH=<some_other_babeld> ./multihop-basic.sh`